databrowser/.prettierrc

```diff
@@ -1,3 +1,6 @@
 {
-    "singleQuote": true
+    "singleQuote": true,
+    "endOfLine": "auto"
+  
+
 }
```

Comment:
i changed this file because when i do npm run lint it gives me error for every end of line (i have windows) should i just restore it and ignore the errors?

Reply:
@d3liaa is this still a problem? In case, I found some information here: https://prettier.io/docs/en/options.html#end-of-line. Maybe we should add `* text=auto eol=lf` to the repo’s `.gitattributes` file? Could you please try that on your windows machine and let me know if it helps?